src/UI/Implementation/Component/Toast/Renderer.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace ILIAS\UI\Implementation\Component\Toast;
+
+use ilException;
+use ILIAS\UI\Component\Button\Shy;
+use ILIAS\UI\Component\Link\Link;
+use ILIAS\UI\Implementation\Render\AbstractComponentRenderer;
+use ILIAS\UI\Implementation\Render\ResourceRegistry;
+use ILIAS\UI\Renderer as RendererInterface;
+use ILIAS\UI\Component;
+use LogicException;
+
+class Renderer extends AbstractComponentRenderer
+{
+    /**
+     * @inheritdoc
+     */
+    public function render(Component\Component $component, RendererInterface $default_renderer) : string
+    {
+        $this->checkComponent($component);
+
+        if ($component instanceof Component\Toast\Toast) {
+            return $this->renderToast($component, $default_renderer);
+        }
+
+        throw new LogicException("Cannot render: " . get_class($component));
+    }
+
+    protected function renderToast(Component\Toast\Toast $component, RendererInterface $default_renderer) : string
+    {
+        $tpl = $this->getTemplate("tpl.toast.html", true, true);
+
+        $title = $component->getTitle();
+        if ($title instanceof Shy || $title instanceof Link) {
+            $title = $default_renderer->render($title);
+        } elseif ($title !== htmlentities($title)) {
+            throw new ilException('Title should not contain HTML!');
+        }
+        $tpl->setVariable("TITLE", $title);
```

Comment:
```suggestion
        $tpl->setVariable("TITLE", htmlentities$title));
```

Please simply render the escaped string instead of throwing an exception here. Same for `$desc`.